README.md

```diff
@@ -1,358 +1,48 @@
-    var lPopResult = conn->lPop("NumberList");
+This connector provides operations for connecting and interacting with Redis server over the network. 
+For more information about configuration and operations, go to the module. 
+- [`ballerinax/redis`](redis/Module.md)
 
-    if (lPopResult is string) {
-        io:println("Popped element: " + lPopResult);
-    } else if (result is ()) {
-        io:println("Key not found");
-    } else {
-        io:println("Error occurred while calling `lPop`");
-    }
-    io:println("===Executing sample set oprerations===");
-    //Sample Set Operations
-    io:println("Adding 3 elements to NumberSet");
-    var setAddResult = conn->sAdd("NumberSet", ["1", "2", "3"]);
+# Building from the Source
+## Setting Up the Prerequisites
 
-    if (setAddResult is int) {
-        io:println(io:sprintf("Number of elements added: : %s", setAddResult));
-    } else {
-        io:println("Error occurred while calling `sAdd`");
-    }
+1. Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).
 
-    io:println("\nQuerying number of elemenets in the Set");
-    var setCardResult = conn->sCard("NumberSet");
+   * [Oracle](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
 
-    if (setCardResult is int) {
-        io:println(io:sprintf("Number of memebers in the set: %s", setCardResult));
-    } else {
-        io:println("Error occurred while calling `sCard`");
-    }
+   * [OpenJDK](https://adoptopenjdk.net/)
 
-    io:println("===Executing sample hash oprerations===");
-    //Sample Hash operations
-    io:println("Adding a key value pair to a hash");
-    var hashSetResult = conn->hSet("HashKey", "Name", "Manuri");
+        > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
 
-    if (hashSetResult is boolean) {
-        io:println(io:sprintf("Hash set status: %s", hashSetResult));
-    } else {
-        io:println("Error occurred while calling `hSet`");
-    }
+2. Download and install [Ballerina Swan Lake Beta2](https://ballerina.io/). 
 
-    io:println("Querying the value of the hash field Name");
-    var hashGetResult = conn->hGet("HashKey", "Name");
+## Building the Source
+Execute the commands below to build from the source.
 
-    if (hashGetResult is string) {
-        io:println("Value of the hash field: " + hashGetResult);
-    } else {
-        io:println("Error occurred while calling `hGet`");
-    }
+1. To build the package:
+    ```    
+    bal build -c ./redis
+    ```
+2. To run the without tests:
+    ```
+    bal build -c --skip-tests ./redis
+    ```
+# Contributing to Ballerina
+As an open source project, Ballerina welcomes contributions from the community. 
 
-    conn.stop();
-    io:println("Redis connection closed!");
-}
+For more information, go to the [contribution guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/main/CONTRIBUTING.md).
```

Comment:
Suggestion: 
"For more information, see the [Contribution Guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/main/CONTRIBUTING.md)."